swagger-v1/swagger-unresolved.yaml

```diff
@@ -1330,6 +1331,41 @@ components:
           example: '#5E6BA3'
       description: |
         The GoCart modal co-branding configuation for the merchant. This is a JSON configuration to control the 'logoTheme', 'buttonTextTheme', 'logo', 'primaryColor' and 'secondaryColor'.
+    MerchantDemoPurchaseDetails:
+      required:
+      - configType
+      properties:
+        configType:
+          type: string
+          example: DemoPurchaseDetails
+          enum:
+          - DemoPurchaseDetails
+        label1:
+          type: string
+          maxLength: 30
+          example: My Merchant
+        description1:
+          type: string
+          maxLength: 30
+          example: 3 items
+        label2:
+          type: string
+          maxLength: 30
+          example: Pickup
+        description2:
+          type: string
+          maxLength: 30
+          example: 25 minutes
+        label3:
+          type: string
+          maxLength: 30
+          example: Store Address
+        description3:
+          type: string
+          maxLength: 105
+          example: 123 main st. Denver, CO 12345
+      description: |
+        Configuration object used to preview purchase details
```

Comment:
@LukeEvans - can this description be made more informative. As someone new to the concept of PurchaseDetails in the api I am a bit confused on what this config is for.

Reply:
@kazarian Updated to the description from teams